frontend/data/locale/en-US.ini

```diff
@@ -956,6 +956,13 @@ Basic.Settings.Appearance.General="General"
 Basic.Settings.Appearance.General.Theme="Theme"
 Basic.Settings.Appearance.General.Variant="Style"
 Basic.Settings.Appearance.General.NoVariant="No Styles Available"
+Basic.Settings.Appearance.FontScale="Font Size"
```

Comment:
Should this key be `Basic.Settings.Appearance.FontSize` or should the string be `Font Scale` @Warchamp7 ? Or is it fine as-is?

Reply:
Font scale is the internal name so I think it's acceptable here